File: CPP/Day-3/factorial.cpp
#include <iostream>
using namespace std;
int main(){
    int n;
    int fact=1;
    cout<<"enter number\n";
    cin>>n;
    for(int i=1;i<=n;i++){
        fact=fact*i;
    }

    cout<<"factorial of no is \t"<<fact;

}

File: WPT/26-sept/demo/src/Factorial.js
import { Component } from "react";

export default class Factorial extends Component {
 
    constructor(props) {
        super(props)  
        this.a = props.num1
    }

    myeventHandler(e) {
        console.log(e.target);
        console.log(e.target.value);
        console.log("in event handler ")

        if (e.target.value == 'Factorial') {
           
        }
        
    }
    render() {
        return (
            <>
                <h1>Factorial of Number</h1>
                Enter a number <input
                    type="text" id="t1"
                    onBlur={(e) => { console.log(e.target.value) }}

                ></input>

                <button value="Factorial" onClick={this.myeventHandler}>
                check
                </button>

            </>
        )

    }


}